### PR TITLE
Mes

### PR DESCRIPTION
**Pull request by manyeyes**
solve:
File "./sherpa/bin/pruned_transducer_statelessX/offline_asr.py", line 298, in decode_waves
TypeError: process() missing 1 required positional argument: 'use_fp16'

### CODE REVIEW

File: sherpa/bin/web/js/offline_record.js [hunk 1]
```diff
@@ -21,8 +21,10 @@ function initWebSocket() {
 
   // Listen for messages
   socket.addEventListener('message', function(event) {
-    document.getElementById('results').value = event.data;
-    console.log('Received message: ', event.data);
+    var result='';
```
Comment (csukuangfj) [inline sherpa/bin/web/js/offline_record.js:24]: Can you remove the changes from this file?

Comment (manyeyes) [reply sherpa/bin/web/js/offline_record.js]: ok，removed.

**Comment by csukuangfj** [comment]
Thanks!

File: sherpa/bin/web/js/offline_record.js (hunk 1 above)
Comment (csukuangfj) [reply sherpa/bin/web/js/offline_record.js]: Looks like you removed the whole file. Could you undo the changes?

Comment (manyeyes) [reply sherpa/bin/web/js/offline_record.js]: Sorry,Undo changes. Look again

Comment (csukuangfj) [reply sherpa/bin/web/js/offline_record.js]: Could you push you new changes?

**Pull request closed without merging**